exercises/exercise-04/index.ts

```diff
@@ -95,9 +95,10 @@ function logPerson(person: Person) {
     console.log(` - ${chalk.green(person.name)}, ${person.age}, ${additionalInformation}`);
 }
 
-function filterUsers(persons: Person[], criteria: User): User[] {
+type Criteria = Partial<Omit<User, 'type'>>;
```

Comment:
I found the hint to use mapped types a bit confusing given that you don't need them.